src/shared/components/plots/downsampleByGrouping.spec.ts

```diff
@@ -76,7 +85,10 @@ describe('downsampleByGrouping', () => {
             {
                 x: 0,
                 y: 4,
-                data: [{ x: 0, y: 4, blah: 'E' }, { x: 1, y: 5, blah: 'F' }],
```

Comment:
https://prettier.io/blog/2019/11/09/1.19.0.html#break-arrays-of-arraysobjects-if-each-element-has-more-than-one-elementproperty-6694-by-sosukesuzuki